tests/testthat/test-epidist_db.R

```diff
@@ -168,6 +168,7 @@ test_that(".read_epidist_db works as expected", {
 })
 
 test_that("sysdata is the same as .read_epidist_db output", {
+  skip_on_covr()
```

Comment:
Can you add a comment explaining why this is necessary so we don't forget please?

Reply:
Resolved in [8ba42c0](https://github.com/epiverse-trace/epiparameter/pull/220/commits/8ba42c0c745d33cca11424c7d3f3070ae75ed9f5). 

I cannot provide a more detailed comment as I could not get to the bottom of what was causing the error. I used `skip_on_covr()` as a fallback for now until I resolve the problem. The error message is relatively uninformative and reading through the {covr} source code it was not clear which point was causing the error (likely [here](https://github.com/r-lib/covr/blob/54de614f5c17a42bfaef2c87dbf5723ca701326a/R/replace.R#L12)).

Reply:
covr works by modifying the functions in your package on-the-fly and roughly speaking adding `covr:::count()` calls to get the number of hits on each line.

So if your tests end up checking the body of a function in the tests, you're going to see this kind of errors.